apps/actions-dependencies-updater/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "actions-dependencies-updater",
+  "version": "0.0.0",
```

Comment:
It'd be good to put an `engine` check in here for `node20`, especially since we'll have others using this script locally.